docs/source/fitting/fitfunctions/ComptonPeakProfile.rst

```diff
@@ -0,0 +1,50 @@
+.. _func-ComptonPeakProfile:
+
+===================
+ComptonPeakProfile
+===================
+
+.. index:: ComptonPeakProfile
+
+Description
+-----------
+
+The ComptonPeakProfile describes the Neutron Compton profile with either a Voigt or Gaussian approximation,
+depending on an energy cutoff value. It takes three input parameters:
+
+-  Intensity: :math:`I`
+-  Position: :math:`P`
+-  SigmaGauss: :math:`\sigma_G`
+
+and three attributes,
+
+-  WorkspaceIndex
+-  Mass
+-  VoigtEnergyCutOff
+
+The VoigtEnergyCutOff is used to determine whether a Gaussian or Voigt function is used to fit the peak. If
+the final energy, as read from the input workspace, is greater than the VoigtEnergyCutOff a normalised Gaussian approximation is used,
+
+.. math:: \frac{I}{2\pi \sigma_T^2}\exp \left( -0.5*\frac{(x-P)^2}{\sigma_T^2} \right)
```

Comment:
Just a matter of preference: perhaps the use of `\cdot` or `\times` would be preferred over the regular multiplication glyph.